.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Asscociated Issue
+ 
+Replace this text with the Jira issue resolved (e.g. CASSANDRA-####) or text specifying why this is a trivial patch.
+
+## Author(s) and Reviewers
+
+Patch by &lt;author>; reviewed by <reviewer[s]>
```

Comment:
needs to be correct to match that found in https://cassandra.apache.org/_/development/how_to_commit.html